packages/arb-bridge-eth/contracts/rollup/NodeGraph.sol

```diff
@@ -192,17 +198,21 @@ contract NodeGraph {
                 prevLeaf,
                 inboxValue,
                 data.afterInboxTop,
-                data.importedMessagesSlice,
-                data.afterVMHash,
-                data.messagesAccHash,
-                data.logsAccHash,
+                data.assertion.inboxHash,
+                data.assertion.afterHash,
```

Comment:
A more descriptive name would be better.  `afterHash` is a hash of what?

Reply:
Updated to be `afterMachineHash`